app/controllers/api/v1/accounts/inbox_members_controller.rb

```diff
@@ -9,7 +9,7 @@ def create
       update_agents_list
       head :ok
     rescue StandardError => e
-      Rails.logger.debug "Rescued: #{e.inspect}"
+      Rails.logger.debug { "Rescued: #{e.inspect}" }
```

Comment:
Instead of this can we use sentry.capture.

Reply:
These are changes from robocop auto-correct. 
Let's take it as a separate task to audit the tracing: https://github.com/chatwoot/chatwoot/issues/2736